File: Confitec-Front/src/app/pages/usuarios/models/request/UpdateUsuario.request.ts
import { EEscolaridade } from "src/app/enums/EEscolaridade";

export class UpdateUsuarioRequest {
  public id: number | undefined;
  public nome: string | undefined;
  public sobrenome: string | undefined;
  public email: string | undefined;
  public dataNascimento: string | undefined;
  public escolaridade: EEscolaridade | undefined;
}

File: Confitec-Back/Confitec.Domain/Models/User/Request/UpdateUsuarioRequest.cs
using Confitec.Domain.Models.Enums;

namespace Confitec.Domain.Models.User.Request
{
    public class UpdateUsuarioRequest
    {
        public int Id { get; set; }
        public string Nome { get; set; } = "";
        public string Sobrenome { get; set; } = "";
        public string Email { get; set; } = "";
        public DateTime DataNascimento { get; set; }
        public EEscolaridade Escolaridade { get; set; }
    }
}
